src/main/java/org/broadinstitute/hellbender/utils/iterators/ReadFilteringIterator.java

```diff
@@ -23,7 +24,7 @@
      * @param nestedIterator underlying iterator from which to pull reads (may not be null)
      * @param readFilter filter to apply to the reads (may not be null)
      */
-    public ReadFilteringIterator( final Iterator<GATKRead> nestedIterator, final ReadFilter readFilter ) {
+    public ReadFilteringIterator( final Iterator<GATKRead> nestedIterator, final Predicate<GATKRead> readFilter) {
```

Comment:
Why was this change necessary?

Reply:
This change got included accidentally from before I refactored the interval filtering code. I was filtering the reads after they had been transformed to GATKReads, and this was a little ergonomic change that allowed using a lambda instead of an anonymous class.